src/app_service/service/community/service.nim

```diff
@@ -476,9 +482,7 @@ QtObject:
     for community in allCommunities:
       self.allCommunities[community.id] = community
 
-    let curatedCommunities = self.loadCuratedCommunities()
-    for curatedCommunity in curatedCommunities:
-      self.curatedCommunities[curatedCommunity.communityId] = curatedCommunity
+    self.asyncLoadCuratedCommunities()
```

Comment:
It would be even better to not even call this on init and only do it when we open the community portal (that's what I meant in the issue as well, sorry if it wasn't clear). Having the call be async is needed and great though, so you already did most of the work.

Reply:
Done